probe-rs/src/architecture/arm/sequences/mod.rs

```diff
@@ -30,6 +42,205 @@ impl DefaultArmSequence {
 
 impl ArmDebugSequence for DefaultArmSequence {}
 
+/// ResetCatchSet for Cortex-A devices
+fn armv7a_reset_catch_set(core: &mut Memory, debug_base: Option<u32>) -> Result<(), crate::Error> {
```

Comment:
Oh, I never realized that these must be the same for all -M or all -A systems :)

Reply:
Some SoCs have slight differences, but these registers / programming patterns are architecturally provided by ARM so they generally work in most cases.